src/main/java/com/redhat/sast/api/service/MlOpsBatchService.java

```diff
@@ -0,0 +1,204 @@
+package com.redhat.sast.api.service;
+
+import java.util.List;
+
+import org.eclipse.microprofile.context.ManagedExecutor;
+
+import com.redhat.sast.api.common.constants.PipelineConstants;
+import com.redhat.sast.api.enums.BatchStatus;
+import com.redhat.sast.api.enums.JobStatus;
+import com.redhat.sast.api.model.Job;
+import com.redhat.sast.api.model.JobBatch;
+import com.redhat.sast.api.platform.PipelineParameterMapper;
+import com.redhat.sast.api.repository.JobBatchRepository;
+import com.redhat.sast.api.v1.dto.request.MlOpsJobCreationDto;
+import com.redhat.sast.api.v1.dto.request.MlOpsJobBatchSubmissionDto;
+import com.redhat.sast.api.v1.dto.response.JobBatchResponseDto;
+
+import io.fabric8.tekton.v1.Param;
+import jakarta.annotation.Nonnull;
+import jakarta.enterprise.context.ApplicationScoped;
+import jakarta.transaction.Transactional;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Service for managing MLOps batch jobs.
+ * Handles submission and processing of MLOps pipeline batches.
+ */
+@ApplicationScoped
+@Slf4j
+public class MlOpsBatchService extends AbstractBatchService {
+
+    private final ManagedExecutor managedExecutor;
+    private final PackageListResolver packageListResolver;
+
+    // No-args constructor required by Quarkus CDI for proxying
+    protected MlOpsBatchService() {
+        super(null, null, null, null, null);
+        this.managedExecutor = null;
+        this.packageListResolver = null;
+    }
+
+    public MlOpsBatchService(
+            JobBatchRepository jobBatchRepository,
+            JobService jobService,
+            PlatformService platformService,
+            PipelineParameterMapper parameterMapper,
+            BatchProcessingService batchProcessingService,
+            ManagedExecutor managedExecutor,
+            PackageListResolver packageListResolver) {
+        super(jobBatchRepository, jobService, platformService, parameterMapper, batchProcessingService);
+        this.managedExecutor = managedExecutor;
+        this.packageListResolver = packageListResolver;
+    }
+
+    /**
+     * Submits an MLOps batch job for processing.
+     */
+    public JobBatchResponseDto submitMlOpsBatch(MlOpsJobBatchSubmissionDto submissionDto) {
+        JobBatch batch = createInitialMlOpsBatch(submissionDto);
+
+        JobBatchResponseDto response = convertToResponseDto(batch);
+
+        managedExecutor.execute(() -> executeMlOpsBatchProcessing(
+                batch.getId(),
+                batch.getUseKnownFalsePositiveFile(),
+                batch.getSubmittedBy(),
+                submissionDto.getDvcNvrVersion(),
+                submissionDto.getDvcKnownFalsePositivesVersion(),
+                submissionDto.getDvcPromptsVersion(),
+                submissionDto.getImageVersion()));
+
+        return response;
+    }
+
+    @Transactional
+    public JobBatch createInitialMlOpsBatch(MlOpsJobBatchSubmissionDto submissionDto) {
```

Comment:
```suggestion
    public JobBatch createJobBatch(MlOpsJobBatchSubmissionDto submissionDto) {
```

Use the same name in the `JobBatchService` class as well